avm/res/fabric/capacity/main.bicep

```diff
@@ -0,0 +1,110 @@
+metadata name = 'Fabric Capacities'
+metadata description = '<Add description>'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the resource to create.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@allowed([
+  'F2'
+  'F4'
+  'F8'
+  'F16'
+  'F32'
+  'F64'
+  'F128'
+  'F256'
+  'F512'
+  'F1024'
+  'F2048'
+])
+@description('Optional. SKU tier of the Fabric resource.')
+param skuName string = 'F2'
```

Comment:
Is it intended to have a default value for this property?

Reply:
Capacity is expensive, so wanted to default to the cheapest SKU. I can remove it if we want to make it a mandatory param.

Reply:
If I may, understood is that defaults in a module should be WAF Alligned, Considering workspace identity and managed private endpoints are a thing from F64 and therefore better candidate being WAF alligned I would suggest make this F64

Reply:
Happy to default to F64, but I'm still concerned about the cost though. F2 is ~ $250/mo, while a F64 is ~ $9300/mo. I wouldn't want someone deploying defaults and get hit with a multi thousand-dollar bill.

Reply:
Let me bring this up with the core team @hundredacres @donheerschap as I believe these are both good points. We use to prefer waf alignment over cost, but I'd like to double check if there should be a trade off in this case

Reply:
Meanwhile, if F64 is the waf compliant one, should that at least be the value for the waf-aligned test? Are workspace identity and managed private endpoint features we can integrate in the module?

Reply:
I have found no WAF documentation for Fabric that would provide guidance on the acceptable sku to use. Also there are no endpoint or identity features available atm.

Reply:
@eriqua There is [this request](https://github.com/Azure/bicep/issues/12905#issuecomment-2268310971) for fabric provider for bicep (as it's not available in ARM).

My suggestion would be either to put the F64 in the WAF e2e test, or make it default and create a minimal e2e test with F2 in it.